bootstrap.sh

```diff
@@ -1 +1,14 @@
 #!/bin/bash
+
+kubectl apply -f .infrastructure/namespace.yml
+kubectl apply -f .infrastructure/statefulSet.yml
+kubectl get all -n mysql
+kubectl apply -f .infrastructure/configMap.yml
+kubectl apply -f .infrastructure/secret.yml
+kubectl apply -f .infrastructure/pv.yml
+kubectl apply -f .infrastructure/pvc.yml
+kubectl apply -f .infrastructure/clusterIp.yml
+kubectl apply -f .infrastructure/nodePort.yml
+kubectl apply -f .infrastructure/hpa.yml
+kubectl apply -f .infrastructure/deployment.yml
```

Comment:
There are not needed deployments included. Clean the list of deployment manifests.